docs/ci-services.md

```diff
@@ -1,45 +1,50 @@
 # CI Services
 
-The CI service blob defines the supported CI services the user will be
+The CI services map defines the supported CI services the user will be
 presented with when running the scaffolder.
 
 ## Example
 
 ```javascript
 {
-  TravisCI: {scaffolder: travisScaffolder, public: true}
+  CIServiceName: {scaffolder: opts => scaffold(opts), public: true}
 }
 ```
 
 ## Config
 
 ### keys: __string__
 
-Keys in the CI services blob define the options the user will be presented with
-when choosing their desired CI service.
+Keys in the CI services map define the choices the user will be presented with
+when choosing their desired CI service. These have no functional impact on the
+scaffolder.
 
-example: `TravisCI`
+example: `CIServiceName`
 
 ### values: __object__
 
-Values in the CI services blob define the actual mechanisms and configuration
+Values in the CI services map define the actual mechanisms and configuration
 that will be used by the JavaScript Scaffolder to scaffold the CI service for
 the user's project.
 
 #### scaffolder: __function__ (_required_)
 
-Consumer-defined plugin(??) called when this ci service is selected by the
-user in the CLI. Will be called with the following arguments:
+Consumer-provided plugin(??) called when this CI service is selected by the
```

Comment:
```suggestion
Consumer-provided plugin called when this CI service is selected by the
```

i'm good with using "plugin" here. might be a good chance to link to an explanation in the glossary once we have one. if we do that, i could see linking the "plugin" text to a more specific term like "CI Service Scaffolder Plugin" or something along those lines